core/src/main/java/io/substrait/expression/Expression.java

```diff
@@ -296,6 +296,7 @@ public ByteString toBytes() {
       ByteBuffer bb = ByteBuffer.allocate(16);
       bb.putLong(value().getMostSignificantBits());
       bb.putLong(value().getLeastSignificantBits());
+      bb.rewind();
```

Comment:
Should this be `bb.flip()`?

Reply:
Good point, I think in this case doesn't matter as we have a 16bytes and put in 2 longs, but generally I think `flip()` is likely the right answer. Updated.

Reply:
BTW I still need to look at how to make the rand() more trackable. I found something in junit to randomize the order of method execution, but nothing super nice/obvious to me to track this rand generation. @jacques-n  do you have a suggestion for that?

Reply:
For now, let's just use a defined seed for all the random inputs so we avoid randomness. We can try to find something else later.